project/cmake/modules/FindLibDvd.cmake

```diff
@@ -76,7 +76,7 @@ if(NOT WIN32)
     endforeach()
 
     set(DVDREAD_CFLAGS "${DVDREAD_CFLAGS} -I${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/libdvd/include")
-    if(KODI_DEPENDSBUILD)
+    if(CMAKE_CROSSCOMPILING)
```

Comment:
Even though it is in the "else" section that only gets executed if KODI_DEPENDSBUILD is not set?